data/blog/new-website.md

```diff
@@ -0,0 +1,191 @@
+to give each app page some visual distinction.
+
+The [apps index page](https://electronjs.org/apps) now also has categories,
+and a keyword filter to find interesting apps like [GraphQL GUIs](https://electronjs.org/apps?q=graphql)
+and [p2p tools](https://electronjs.org/apps?q=graphql).
+
+If you've got an Electron app that you'd like featured on the site, open a 
+pull request on the [electron/electron-apps] repository.
+
+## One-line Installation with Homebrew
+
+The [Homebrew] package manager for macOS has a subcommand called [cask]
+that makes it easy to install desktop apps using a single command in your
+terminal, like `brew cask install atom`.
+
+We've begun collecting Homebrew cask names for popular Electron apps and are now
+displaying the installation command (for macOS visitors) on every app page 
+that has a cask:
+
+<figure>
+  <a href="https://electronjs.org/apps/dat">
+   <img  class="screenshot" src="https://user-images.githubusercontent.com/2289/32871246-c5ef6f2a-ca34-11e7-8eb4-3a5b93b91007.png">
+   <figcaption>Installation options tailored for your platform: macOS, Windows, Linux</figcaption>
+  </a>
+</figure>
+
+To view all the apps that have homebrew cask names, visit 
+[electronjs.org/apps?q=homebrew](https://electronjs.org/apps?q=homebrew). If 
+you know of other apps with casks that we haven't indexed yet, 
+[please add them!](https://github.com/electron/electron-apps/blob/master/contributing.md)
+
+## 🌐 A New Domain
+
+We've moved the site from electron.atom.io to a new domain: [electronjs.org].
+
+The Electron project was born inside [Atom], GitHub's open-source text editor 
+built on web technologies. Electron was originally called `atom-shell`. Atom 
+was the first app to use it, but it didn't take long for folks to realize that 
+this magical Chromium + Node runtime could be used for all kinds of different 
+applications. When companies like Microsoft and Slack started to make use of
+`atom-shell`, it became clear that the project needed a new name.
+
+And so "Electron" was born. In early 2016, GitHub assembled a new team to focus
+specifically on Electron development and maintenance, apart from Atom. In the 
+time since, Electron has been adopted by thousands of app developers, and is now
+depenended on by many large companies, many of which have Electron teams of 
+their own.
+
+Supporting GitHub's Electron projects like Atom and [Desktop] is still a 
+priority for our team, but by moving to a new domain we hope to help clarify 
+the technical distinction between Atom and Electron.
+
+## 🐢🚀 Node.js Everywhere
+
+The previous Electron website was built with [Jekyll], the popular Ruby-based 
+static site generator. Jekyll is a great tool for building static websites, but 
+the website had started to outgrow it. We wanted more dynamic capabilities like proper redirects and dynamic content rendering, so a Node.js server was the obvious choice.
```

Comment:
Should this be `[Node.js]` with a link below, just as we've done for `[Jekyll]` et al?